packages/zipkin/index.d.ts

```diff
@@ -0,0 +1,159 @@
+declare namespace zipkin {
```

Comment:
Maybe make a comment when to change this? Ex whenever we change any file?

Reply:
I will add a comment.